server/neptune/temporalworker/job/partition_registry_test.go

```diff
@@ -133,7 +133,7 @@ func TestPartitionRegistry_Register(t *testing.T) {
 		}
 
 		buffer := make(chan string, 100)
-		go partitionRegistry.Register(jobID, buffer)
+		go partitionRegistry.Register(context.TODO(), jobID, buffer)
```

Comment:
Tests should never have context.TODO() because that implies we will add it (Which we won't cuz they're tests), just use background context